docs/content/en/docs/configuration/keys.md

```diff
@@ -2855,16 +2856,20 @@ See also:
 | Configuration key | Scope  | Default | Since |
 |-------------------|--------|---------|-------|
 | `waf`             | `Path` |         |       |
+| `waf-fail-closed` | `Path` | `true`  | v0.15 |
 | `waf-mode`        | `Path` | `deny`  | v0.9  |
 
+
 Defines which web application firewall (WAF) implementation should be used
 to validate requests. Currently the only supported value is `modsecurity`.
 
 This configuration has no effect if the ModSecurity endpoints are not configured.
 
+The `waf-fail-closed` key determines whether an error from the WAF (such as a timeout) should
+cause the request to be denied ("true") or allowed ("false"). The default value of "true" means that errors or timeouts will cause requests to be denied.
```

Comment:
add something like "true since v0.15". I'll update this doc and the default value in v0.14 to false.

Reply:
I think I understand what you're saying: v0.14 will default to false, and v0.15 will default to true, presumably to make it easier for users to migrate. Does my updated sentence look okay now?

Reply:
Sure it's fine, thanks!